recipe/run_conda_forge_build_setup_win

```diff
@@ -1,3 +1,8 @@
+
+:: 2 cores available on Appveyor workers: https://www.appveyor.com/docs/build-configuration/#build-environment
```

Comment:
Fixing in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/73 ).